app/epa/settings.py

```diff
@@ -1,265 +1,265 @@
-LOGOUT_REDIRECT_URL = "home"
-
-CRISPY_TEMPLATE_PACK = "bootstrap4"
-
-# Please note, we don't use Django's internal email system,
-# we implement our own, using exchangelib
-USE_EXCHANGE_EMAIL_BACKEND = env.bool("USE_EXCHANGE_EMAIL_BACKEND", default=True)
-
-# The Exchange account which sends emails
-EXCHANGE_ACCOUNT = env("EXCHANGE_ACCOUNT", default="dummy@dummy.com")
-EXCHANGE_PW = env("EXCHANGE_PW", default="dummypw")
-EXCHANGE_EMAIL = env("EXCHANGE_EMAIL", default="dummy@dummy.com")
-EXCHANGE_SERVER = env("EXCHANGE_SERVER", default="dummy.com")
-# Email addresses to which feedback emails will be sent
-RECIPIENTS = env.list("RECIPIENTS", default=["dummy@dummy.com", "dummy2@dummy.com"])
-EMAIL_SUBJECT_PREFIX = env("EMAIL_SUBJECT_PREFIX", default="[open_plan] ")
-
-MESSAGE_TAGS = {
-    messages.DEBUG: "alert-info",
-    messages.INFO: "alert-info",
-    messages.SUCCESS: "alert-success",
-    messages.WARNING: "alert-warning",
-    messages.ERROR: "alert-danger",
-}
-
-USE_PROXY = env.bool("USE_PROXY", default=True)
-PROXY_ADDRESS_LINK = env("PROXY_ADDRESS", default="http://proxy:port")
-PROXY_CONFIG = (
-    ({"http://": PROXY_ADDRESS_LINK, "https://": PROXY_ADDRESS_LINK})
-    if USE_PROXY
-    else (dict())
-)
-
-MVS_API_HOST = env("MVS_API_HOST", default="https://mvs-eland.rl-institut.de")
-MVS_POST_URL = f"{MVS_API_HOST}/sendjson/"
-MVS_GET_URL = f"{MVS_API_HOST}/check/"
-MVS_LP_FILE_URL = f"{MVS_API_HOST}/get_lp_file/"
-MVS_SA_POST_URL = f"{MVS_API_HOST}/sendjson/openplan/sensitivity-analysis"
-MVS_SA_GET_URL = f"{MVS_API_HOST}/check-sensitivity-analysis/"
-
-# Allow iframes to show in page
-X_FRAME_OPTIONS = "SAMEORIGIN"
-
-LOGGING = {
-    "version": 1,
-    "disable_existing_loggers": False,
-    "formatters": {
-        "dtlnm": {
-            "format": "%(asctime)s - %(levelname)8s - %(name)s - %(message)s",
-            "datefmt": "%Y-%m-%d %H:%M:%S",
-        }
-    },
-    "handlers": {
-        "info_file": {
-            "level": "INFO",
-            "class": "logging.FileHandler",
-            "filename": "django_epa_info.log",
-            "formatter": "dtlnm",
-        },
-        "warnings_file": {
-            "level": "WARNING",
-            "class": "logging.FileHandler",
-            "filename": "django_epa_warning.log",
-            "formatter": "dtlnm",
-        },
-        "console": {
-            "level": "WARNING",
-            "class": "logging.StreamHandler",
-            "stream": sys.stdout,
-        },
-    },
-    "loggers": {
-        "": {
-            "handlers": ["info_file", "warnings_file", "console"],
-            "level": "DEBUG",
-            "propagate": True,
-        },
-        "asyncio": {"level": "WARNING"},
-    },
-}
-
-# DJANGO-Q CONFIGURATION
-# source: https://django-q.readthedocs.io/en/latest/configure.html
-Q_CLUSTER = {
-    "name": "django_q_orm",
-    "workers": 4,
-    "timeout": 90,
-    "retry": 120,
-    "queue_limit": 50,
-    "orm": "default",
-}
+"""
```

Comment:
Actually this change could be perfomed projectwide in a single commit in another PR. 
How to make sure then that windows user do not reintroduce CRLF in the future?